activerecord/lib/active_record/connection_adapters/sqlite3/schema_creation.rb

```diff
@@ -5,6 +5,14 @@ module ConnectionAdapters
     module SQLite3
       class SchemaCreation < SchemaCreation # :nodoc:
         private
+          def visit_TableDefinition(o)
+            create_sql = super
+            if ConnectionAdapters::SQLite3Adapter.strict_tables && o.strict_table
```

Comment:
Shouldn't `ConnectionAdapters::SQLite3Adapter.strict_tables` acts as the default value rather than needed both that global config and the table to set strict mode?

Reply:
I've tried setting the global config and the table to strict mode, as the default values affect the current table.
I think it should be the default in the future, but I think it's also a good option to select it as an option.

Should it be rewritten to default as described in the issue?